### PR TITLE
Add support for Edge Legacy vs. Edge

### PR DESCRIPTION
**Pull request by aaronshekey**
👋 This PR bumps some data dependencies and splits Edge and Edge Legacy.

image

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ if (app.get("env") === "development") {
     res.status(err.status || 500);
     res.render("error", {
       message: err.message,
-      error: err
+      error: err,
```
Comment (ai) [inline app.js:41]: Why do you updated code styles?

Comment (aaronshekey) [reply app.js:41]: `npm run test` reported those as failures. Happy to revert!

Comment (ai) [reply app.js:41]: Can be your revert these changes. I already spend all my free time for preparing PostCSS 8 release?

Comment (aaronshekey) [reply app.js]: Reverted 😀

**Comment by ai** [comment]
Thanks for the fixes. I am worry about the warnings from Now.

**Comment by aaronshekey** [comment]
Stack Overflow is currently using this website as an amazing visualizer for the browsers we support. I would love to get it updated.

Looks like every PR is broken in the same fashion. Not sure how to fix the Vercel configuration as an outside collaborator. There were a few commits trying to fix it, right?

@jonrohan Do you have any insight there? Also, do you want me to add the fixes the linter's suggesting?

**Comment by ai** [comment]
I have a plan to refactor the website to fix many problems like building problems you are seeing here.

We are collecting funds to find time for this work (together with security refactoring it will take few months)

https://opencollective.com/browserslist

**Pull request closed without merging**